newrelic/resource_newrelic_alert_policy.go

```diff
@@ -78,6 +108,12 @@ func resourceNewRelicAlertPolicyRead(d *schema.ResourceData, meta interface{}) e
 		return err
 	}
 
+	channelIDs := d.Get("channel_ids").([]interface{})
```

Comment:
Fairly certain we don't need to do this here because state is still preserved due to the fact that if a user adds or removes any channel IDs, the original resource is destroyed and then recreated with the updated channel associations (via the `ForceNew: true` schema option). @ctrombley @zlesnr any thoughts on that?